client/README.md

```diff
@@ -1,29 +1,51 @@
-# client
-
-## Project setup
-```
-npm install
-```
-
-### Compiles and hot-reloads for development
-```
-npm run serve
-```
-
-### Compiles and minifies for production
-```
-npm run build
-```
-
-### Run your unit tests
-```
-npm run test:unit
-```
-
-### Lints and fixes files
-```
-npm run lint
-```
-
-### Customize configuration
-See [Configuration Reference](https://cli.vuejs.org/config/).
+# TheHaloMod-SPA Client
+
+[![Github Actions Client CI Workflow](https://img.shields.io/github/workflow/status/halomod/TheHaloMod-SPA/Client%20CI?label=Client%20CI)](https://github.com/halomod/TheHaloMod-SPA/actions/workflows/client.yaml)
+
+## Table of Contents
+
+- [Usage](#usage)
+  - [How to run the client locally](#how-to-run-the-client-locally)
+  - [How to build and run the production version of the client locally](#how-to-build-and-run-the-production-version-of-the-client-locally)
+  - [Development commands](#development-commands)
+- [Architecture](#architecture)
+
+## Usage
+
+### How to run the client locally
+
+1. Make sure you are in the `TheHaloMod-SPA/client` directory in your shell of choice
+1. Run `npm install`. (If you don't have npm installed, [see here for instructions](https://github.com/nvm-sh/nvm#installing-and-updating)).
+1. Run `npm run serve`
+1. The client should be available on `localhost:8080`
+
+### How to build and run the production version of the client locally
+
+1. Make sure you are in the `TheHaloMod-SPA/client` directory in your shell of choice
+1. Run `npm install` if that hasn't already been done. (If you don't have npm installed, [see here for instructions](https://github.com/nvm-sh/nvm#installing-and-updating)).
+1. Run `npm run build` and wait for it to finish
+1. Run `serve dist -p 5111`. Note that `-p 5111` sets the port to `5111` because by default the port is set to `5000` which the server is running on. So `5111` can be any port that you choose that isn't already taken. Also if `serve` isn't installed, [see here for instructions](https://www.npmjs.com/package/serve).
```

Comment:
Should probably actually number these instead of just all ones lol

Reply:
@Tengelma The reason it is done that way is so if a step needs to be moved around they don't all need to be re-numbered again. This way it is left to the renderer to do it's thang. With that said, do you still want it to be numbered in the code?

Reply:
Ah, didn't realize renderer just handled it. We're all good then. Nice work!

Reply:
Rock on. Thanks for taking a look 😄